musicbot/commands/music.py

```diff
@@ -160,6 +163,21 @@ async def _stop(self, ctx):
         await utils.guild_to_audiocontroller[current_guild].stop_player()
         await ctx.send("Stopped all sessions :octagonal_sign:")
 
+    @commands.command(name='move', description=config.HELP_MOVE_LONG, help=config.HELP_MOVE_SHORT, aliases=['mv'])
+    async def _move(self, ctx, oldindex: int, newindex: int):
```

Comment:
I think it would be a better idea to use `*args` and validate them so a missing argument exception is not thrown when run blank. Also should output some kind of error when run blank.